easy-ui-tokens/src/theme/light.json

```diff
@@ -1,24 +1,37 @@
 {
   "theme.light": {
```

Comment:
the only thing i don't feel great about with the theming work is having the theme tokens defined both here and in the `Theme.tsx` file.

the `Theme.tsx` file is the source of truth for the theme tokens in our Easy UI project, and the tokens defined here our the specific implementation of that truth for EasyPost branding, but it still feels a little clunky having both. it may be simpler to use the tokens defined here as the truth instead, but i'm not sure if we can make that type safe. i may look into that.

any ideas are welcome.

Reply:
looks like it is in fact possible to derive the theme definition types from the tokens. i think i like this approach better

Reply:
agree